README.md

```diff
@@ -1 +1,56 @@
-# Template3-project.
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Template3 project.
+
+> This is project for template3 portfolio.
```

Comment:
- [ ] Please edit this to be a descriptive title and give it a meaningful description under it.